rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeRequest.groovy

```diff
@@ -64,6 +64,8 @@ class BakeRequest {
   String template_file_name
   @ApiModelProperty("A map of key/value pairs to add to the packer command")
   Map extended_attributes
+  @ApiModelProperty("The name of a json file containing key/value pairs to add to the packer command")
```

Comment:
It's not an absolute path. It's treated the same as the template file name just above. I've amended the comment to make that clear.